enginetest/enginetests.go

```diff
@@ -185,33 +197,29 @@ func TestReadOnly(t *testing.T, harness Harness) {
 	}
 
 	for _, query := range writingQueries {
-		_, _, err = e.Query(NewCtx(idxReg), query)
+		_, _, err = e.Query(NewContext(harness), query)
 		require.Error(err)
 		require.True(auth.ErrNotAuthorized.Is(err))
 	}
 }
 
 func TestExplode(t *testing.T, harness Harness) {
 	db := harness.NewDatabase("mydb")
-	table := harness.NewTable(db, "t", sql.Schema{
+	table, err := harness.NewTable(db, "t", sql.Schema{
 		{Name: "a", Type: sql.Int64, Source: "t"},
 		{Name: "b", Type: sql.CreateArray(sql.Text), Source: "t"},
 		{Name: "c", Type: sql.Text, Source: "t"},
 	})
+	require.NoError(t, err)
 
-	InsertRows(
-		t, mustInsertableTable(t,table),
-		sql.NewRow(int64(1), []interface{}{"a", "b"}, "first"),
-		sql.NewRow(int64(2), []interface{}{"c", "d"}, "second"),
-		sql.NewRow(int64(3), []interface{}{"e", "f"}, "third"),
-	)
+	InsertRows(t, harness.NewContext(), mustInsertableTable(t, table), sql.NewRow(int64(1), []interface{}{"a", "b"}, "first"), sql.NewRow(int64(2), []interface{}{"c", "d"}, "second"), sql.NewRow(int64(3), []interface{}{"e", "f"}, "third"), )
```

Comment:
I prefer the old list format instead but it's nbd

Reply:
yeah me too, my editor did that and I was too lazy to undo it